### PR TITLE
Hide upload button

### PR DESCRIPTION
**Pull request by jameseaster**
#### _Resolves - offers ability to hide `Upload` image button_

---

### What kind of change does this PR introduce?

- UI customization.

<br/>

### What is the current behavior?

- Camera and Upload button are always present.

<br/>

### What is the new behavior?

- Upload button can be hidden with a prop.

<br/>

### Does this PR introduce a breaking change?

- No

<br/>

### How has this been tested?

- Symlinked and locally rendered.

<br/>

### Screenshots: n/a

### CODE REVIEW

File: components/BcsButtons.vue [hunk 1]
```diff
@@ -61,6 +63,10 @@ export default {
       type: Boolean,
       default: undefined
     },
+    hideUploadButton: {
+      type: Boolean,
+      default: undefined
```
Comment (mattcollier) [inline components/BcsButtons.vue:68]: if the `type` is boolean, would a type of `false` be more appropriate?

**Comment by mattcollier** [comment]
@jameseaster there is a linting warning shown in the files tab, can that be addressed?

File: components/BcsButtons.vue (hunk 1 above)
Comment (davidlehn) [reply components/BcsButtons.vue:68]: The BarcodeScanner one was updated, but this BcsButtons one should be too I assume.

**Review by dlongley — APPROVED** [review]

File: components/BcsButtons.vue (hunk 1 above)
Comment (dlongley) [inline components/BcsButtons.vue:68]:
```suggestion
      default: false
```

**Pull request merged**